source/_integrations/apcupsd.markdown

```diff
@@ -20,22 +20,38 @@ ha_integration_type: integration
 
 [apcupsd](http://www.apcupsd.org/) status information can be integrated into Home Assistant when the Network Information Server (NIS) is configured on the APC device.
 
-There is currently support for the following device types within Home Assistant:
+## Prerequisites
 
-- [Binary sensor](#binary-sensor)
-- [Sensor](#sensors)
+1. **Install apcupsd**
 
-## Home Assistant add-on installation
+    First, install [apcupsd](http://www.apcupsd.org/) on the machine connected to your UPS. It works on Linux, macOS, Windows, BSD, Solaris, and more.  
+    You can usually install it through your operating system’s package manager. For example:
 
-Install this [unofficial add-on](https://github.com/korylprince/hassio-apcupsd/) to use this integration with Home Assistant. Keep in mind that we can't give you support for this add-on.
+    - **Ubuntu/Debian**: `sudo apt install apcupsd`
+    - **Red Hat/Fedora**: `sudo dnf install apcupsd`
+    - **macOS** (with Homebrew): `brew install apcupsd`
+    - **Windows**: Download the installer from the [apcupsd website](http://www.apcupsd.org/)
```

Comment:
```suggestion
```
I think this is too much detail. I see the convenience, but the actual steps for installing apcupsd are out of scope for this documentation.